listener/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(listener)
+
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_STANDARD_REQUIRED ON)
+endif()
+
+find_package(catkin REQUIRED COMPONENTS
+  rxros
+  std_msgs
+)
+
+catkin_package(
+   CATKIN_DEPENDS 
+     rxros std_msgs
+)
+
+include_directories(
+  ${catkin_INCLUDE_DIRS}
+)
+
+link_directories(${catkin_LIBRARY_DIRS})
```

Comment:
This should not be necessary. Did you add this yourself?

Reply:
removed. no idea from where it came.  works without. thanks :)